src/ontology/uberon-edit.obo

```diff
@@ -221690,30 +221690,27 @@ relationship: dc-contributor https://orcid.org/0000-0003-2699-3066
 
 [Term]
 id: UBERON:7500036
-name: line from caundal-sacral region of the vertebral column to the last vertebra
-def: "An anatomical line along a tail that extends from the first caudal vertebra to the last cuadal vertebra." [https://orcid.org/0000-0003-2699-3066]
-intersection_of: UBERON:0006800 ! anatomical line
-intersection_of: connects UBERON:0011271 ! caudal-sacral region of vertebral column
-intersection_of: connects UBERON:7500035 ! tip of tail
-relationship: dc-contributor https://orcid.org/0000-0003-2699-3066
+name: obsolete line from caundal-sacral region of the vertebral column to the last vertebra
+def: "OBSOLETE An anatomical line along a tail that extends from the first caudal vertebra to the last cuadal vertebra." [https://orcid.org/0000-0003-2699-3066]
+comment: No longer part of the pattern design for lengths of lines. Can be found in OBA as tail length (http://purl.obolibrary.org/obo/OBA_VT0002758).
+is_obsolete: true
+consider: http://purl.obolibrary.org/obo/OBA_VT0002758
```

Comment:
One more minor change is needed:
the full URI for the consider should be added in the IRI editor, not in the Literal tab.

here is a screenshot:
<img width="842" alt="image" src="https://user-images.githubusercontent.com/6722114/159799559-4bb8d7d3-6fc9-4a9b-8717-c72cdabadf36.png">

Could you please fix these?

thanks!

Reply:
the diff should look like this: 
consider: OBA:VT0002758